packages/next/client/future/image.tsx

```diff
@@ -228,13 +228,13 @@ function generateImgAttrs({
 }
 
 function getInt(x: unknown): number | undefined {
-  if (typeof x === 'number') {
+  if (typeof x === 'number' || typeof x === 'undefined') {
```

Comment:
Are these checks needed, doesn't `parseInt()` tolerate non-number values being passed to it already and returns `NaN`, also should we be defining the base when calling `parseInt(value, 10)`?